conda_vendor/conda_vendor.py

```diff
@@ -33,38 +33,42 @@ def create_vendored_dir(environment_file, platform, desired_path=None) -> Path:
             sys.exit(f"Failed to read environment name from {environment_file}")
     
     # use current working directory if no path specified
+
+    def _create_vendored_dir(root_dir, env_name, platform):
+
+        if isinstance(root_dir, str):
+            root_dir = Path(root_dir) 
+
+        path = root_dir / env_name
+        os.mkdir(path)
```

Comment:
You can use `pathlib.Path.mkdir` for this. Also, I think maybe a check to see if the path exists, and if it does, clean out that directory (or delete and recreate). Having to do this manually is frustrating. Log an info statement saying that you delete and recreated.

Reply:
Good catch!